powerline/config_files/themes/powerline_unicode7.json

```diff
@@ -143,6 +143,15 @@
 			"args": {
 				"text": "🖫⃥"
 			}
+		},
+
+		"powerline.segments.i3wm.scratchpad": {
+			"args": {
+				"icons": {
+					"fresh": "O",
+					"changed": "X"
```

Comment:
wait, is this file for `UTF-7` or for `Unicode 7.0` symbols? Either way almost none of the other symbols are displayed correctly in vim (or in my browser) so I assumed that if I pasted mine in and they showed up that would mean they are wrongly encoded.